Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM golang:1.6
 
-RUN go get -u github.com/segment-sources/postgres
 
-# Additionally add a cron-like runner to run on an
-# interval.
-RUN go get -u github.com/segmentio/go-every
+ADD . /go/src/github.com/segment-sources/postgres
```

Comment:
no idea where i should be putting this. @tejasmanohar is there like a convention i should be using here?